h2o-algos/src/main/java/hex/ensemble/Metalearners.java

```diff
@@ -227,8 +227,8 @@ protected void setCustomParams(GLMParameters parms) {
             parms._generate_scoring_history = true;
             parms._score_iteration_interval = (parms._valid == null) ? 5 : -1;
 
-            //specific to AUTO mode
-            parms._non_negative = true;
+            //specific to AUTO mode; non_negative constraint is not supported for multinomial tasks
+            parms._non_negative = _parms.train().vec(_parms.getResponseColumn()).cardinality() <= 2;
```

Comment:
GLM can fail in SE when using beta/non_negative constraints with multinomial classification (currently unsupported by GLM).

Reply:
It does not support Ordinal either.